x-pack/plugins/kubernetes_security/public/components/container_name_widget/styles.ts

```diff
@@ -47,14 +50,34 @@ export const useStyles = () => {
     };
 
     const countValue: CSSObject = {
-      fontWeight: font.weight.semiBold,
+      fontSize: size.m,
+      minWidth: '76px',
```

Comment:
for consistency, it should follow the same width as the Column Header (so you can either remove it or add the same property to the column header)